batchlib/analysis/cell_level_analysis.py

```diff
@@ -604,6 +629,112 @@ def validate_outputs(self, output_files, folder, status, ignore_failed_outputs):
             return have_table
 
 
+def add_site_name_to_image_table(column_names, table):
+    assert column_names[0] == 'image_name'
+    column_names = list(column_names)
+    column_names.insert(1, 'site_name')
+    table = np.array(table).tolist()
+    for row in table:
+        row.insert(1, image_name_to_site_name(row[0]))
+    return column_names, np.array(table)
+
+
+class ExtractBackground(CellLevelAnalysisWithTableBase):
```

Comment:
The `cell_level_analysis.py` file is getting a bit long, should we maybe refactor `InstanceFeatureExtraction` and this class to a new file, maybe `cell_level_features.py`?

Reply:
Yes, we should do that at some point soon.